django/views/decorators/debug.py

```diff
@@ -41,6 +46,34 @@ def sensitive_variables_wrapper(*func_args, **func_kwargs):
                 sensitive_variables_wrapper.sensitive_variables = "__ALL__"
             return func(*func_args, **func_kwargs)
 
+        if iscoroutinefunction(func):
+            markcoroutinefunction(sensitive_variables_wrapper)
```

Comment:
I think that defining `async def sensitive_variables_wrapper()` (as in the previous version) is preferable over using `markcoroutinefunction()`, and we would avoid defining unused `sensitive_variables`:
```python
        if iscoroutinefunction(func):
            @wraps(func)
            async def sensitive_variables_wrapper(*func_args, **func_kwargs):
                return await func(*func_args, **func_kwargs)

           ...
```

Reply:
Sounds good to me. I was more focused on reducing code duplication but this LGTM!